docker-compose.yml

```diff
@@ -1,44 +1,46 @@
 version: "3"
 services:
   backend:
-    build: .
+    image: openstatefoundation/openraadsinformatie-backend
     volumes:
       - .:/opt/ori
     links:
-      - celery
-      - elasticsearch
-    #command: "/opt/ori/start.sh"
-    command: "tail -f /dev/null"
+      - redis:redis
+      - elasticsearch:elasticsearch
   frontend:
-    build:
-      context: .
-      dockerfile: Dockerfile.frontend
-    volumes:
-      - .:/opt/ori
+    image: openstatefoundation/openraadsinformatie-frontend
     links:
-      - elasticsearch
+      - elasticsearch:elasticsearch
     ports:
       - "5000:5000"
   redis:
     image: "redis:${REDIS_VERSION}"
   elasticsearch:
```

Comment:
The elasticsearch entry in docker-compose has yet to be optimized for production.
See https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html